test/Digipolis.ServiceAgents.UnitTests/BaseClass/AgentBaseTests.cs

```diff
@@ -120,7 +120,7 @@ public class AgentBaseTests : ServiceAgentTestBase
         }
 
         [Fact]
-        public async Task JsonParserErrorWithoutParams()
+        public async Task JsonParserErrorWithMissingTitleAndStatusCode()
```

Comment:
De aangepaste test (JsonParserErrorWithoutParams) slaagt niet meer. Dit betekent dat de body van de error message niet meer in Messages beschikbaar is. De toepassing kan de echte fout dus niet meer achterhalen. Lijkt mij wel iets dat nuttig was?

Reply:
Zoals besproken: dit blijkt toch te werken: door throw Exception te doen in AgentBase is de body nog steeds beschikbaar.